docs/src/css/custom.css

```diff
@@ -4,6 +4,10 @@
  * work well for content-centric websites.
  */
 
+@tailwind base;
+@tailwind components;
+@tailwind utilities;
```

Comment:
Seems like we are using tailwind now. Can you update the documentation above to mention this and how this relates to the "infima" CSS framework? Maybe one is used for the custom landing page while the other is used by docusaurus?

Reply:
Sure thing - I'll provide some documentation on it

Reply:
Leaving this unresolved as I can't seem to find that docs?

Reply:
We'll live without it.